packages/app/src/admin/ProjectAdminConfigPage.tsx

```diff
@@ -0,0 +1,39 @@
+import { Button, Divider, PasswordInput, Stack, TextInput, Title } from '@mantine/core';
+import { showNotification } from '@mantine/notifications';
+import { forbidden, normalizeErrorString } from '@medplum/core';
+import { Document, Form, OperationOutcomeAlert, useMedplum } from '@medplum/react';
+
+export function ProjectAdminConfigPage(): JSX.Element {
+  const medplum = useMedplum();
+
+  if (!medplum.isLoading() && !medplum.isProjectAdmin()) {
+    return <OperationOutcomeAlert outcome={forbidden} />;
+  }
+
+  function forceSetPassword(formData: Record<string, string>): void {
+    medplum
+      .post('admin/projects/setpassword', formData)
+      .then(() => showNotification({ color: 'green', message: 'Done' }))
+      .catch((err) => showNotification({ color: 'red', message: normalizeErrorString(err), autoClose: false }));
+  }
+
+  return (
+    <Document width={600}>
+      <Title order={1}>Project Admin</Title>
+      <Divider my="lg" />
+      <Title order={2}>Force Set Password</Title>
+      <p>
+        Note that this applies to all projects for the user. Therefore, this should only be used in extreme
+        circumstances. Always prefer to use the "Forgot Password" flow first.
+      </p>
+      <Form onSubmit={forceSetPassword}>
+        <Stack>
+          <TextInput name="email" label="Email" required />
+          <PasswordInput name="password" label="Password" required />
+          <TextInput name="projectId" label="Project ID" />
```

Comment:
I think we can remove the `projectId` input, because it is implicit in this case, correct?